### DISCUSSION TITLE
Mac OS Terminal Green Error

### DISCUSSION THREAD

**Discussion by samuelswandi** [NvChad/NvChad#1939] [Q&A] [answered]
Hi, I'm wondering on how to fix this issue on my mac terminal. I've installed neovim, changed the font to nerd font through terminal settings, and install nvchad. Thanks!

<img width="571" alt="Screenshot 2023-04-18 at 12 09 18" src="https://user-images.githubusercontent.com/79347993/232676825-66c6cd30-6380-46b4-bc62-c761101a8a56.png">

**Comment by siduck** [comment, marked as answer]
hi @samuelswandi , macos default terminal doesnt support true colors. so pls use iterm2 or alacritty/wezterm/kitty

**Comment by Nandakyawwin** [comment]
hi @samuelswandi ,I also face that probem,I used WezTerm terminal.

**Reply by siduck** [reply]
@Nandakyawwin  terminal issue , please use true colors

**Reply by samuelswandi** [reply]
Yup, I solved it using iterm2

**Reply by Nandakyawwin** [reply]
Change the terminal app in mac,I solved.thanks you
